h2/events.py

```diff
@@ -212,6 +267,11 @@ def __init__(self):
         #: the changed settings.
         self.changed_settings = {}
 
+    def __repr__(self):
+        return "<SettingsAcknowledged changed_settings:%s>" % (
+            self.changed_settings,
```

Comment:
Ok, but then the name is misleading. `self.acknowledged_settings` seems more intuitive to me.